app/views/declaration/commodityMeasure/commodity_measure_default.scala.html

```diff
@@ -16,13 +16,44 @@
 
 @import forms.declaration. CommodityMeasure
 @import views.html.components.gds._
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import config.AppConfig
 
 @this(
         inputText: exportsInputText,
-        inputTextUnits: exportsInputTextWithPrefix
+        inputTextUnits: exportsInputTextWithPrefix,
+        exportsInputCheckbox: exportsInputCheckbox,
+        govukDetails : GovukDetails,
+        govukHint: GovukHint,
+        link: link,
+        appConfig: AppConfig
 )
 
-@(form: Form[CommodityMeasure])(implicit messages: Messages)
+@(form: Form[CommodityMeasure], maybeCommodityCode: Option[String])(implicit messages: Messages)
+
+@expanderContent = {
+    @maybeCommodityCode.map { commodityCode =>
```

Comment:
Could you maybe calling `getCommodityCode` passing `request.cacheModel` avoid to pass `maybeCommodityCode` from the controller?

Reply:
I prefer passing the smallest amount of info required (i.e the optional commodity code, rather than the whole ExportDeclaration class), make testing much easier.

Reply:
Its interesting. I don't actually have a implicit `request` arg in this view class (but I do in the `commodity_measure.scala.html` that calls this view as you point out below). I have played with removing it as you suggest and while it does simplify the controller (that I like) it does complicates the tests quite a bit (as instead of simply passing the `commodityCode` to the view, we now have to craft a request instance and a whole `ExportsDeclaration` model. 

I'd prefer to leave as is for this PR, though I'll keep this in mind for future :+1: